meshtastic/config.proto

```diff
@@ -345,6 +346,39 @@ message Config {
       NOT_PRESENT = 2;
     }
 
+    /*
+     * A plain language label that users of GPS select based on their intended activity.
+     * We use this to make selections from the variety of GPS Settings, including:
+     * GPS Update Interval, Position Broadcast Interval, Smart Broadcast Min Interval,
+     * Smart Broadcast Min Distance, and Position Flags. The firmware also modifies the
+     * GPS hardware configuration to take advantage of advanced features for each case.
+     * The firmware may also discard erroneous positions based on movement determined to
+     * be physically impossible.
+     */
+    enum GpsProfile {
+      /*
+       * Profile not set, user will make their own decisions.
+       */
+      MANUAL = 0;
+
+      /*
+       * This node does not move.
+       * We automatically apply this to all routers, router_lates,
+       * and any node with manually set lat/lon.
+       */
+      FIXED_POSITION = 1;
+
+      /*
+       * Hikers, runners, walkers.
+       */
+      PEDESTRIAN = 2;
+
+      /*
+       * Cars, motorbikes.
+       */
+      VEHICLE = 3;
+    }
```

Comment:
Can we add airborn? People are experimenting with Drones, Kites and Ballons a lot.